webpack.config.js

```diff
@@ -1,7 +1,7 @@
 const path = require("path");
 
 module.exports = {
-  entry: __dirname + "/src/root-config-dist.js",
+  entry: __dirname + "/src/root-config.js",
```

Comment:
The entry file was root-config-dist.js, but the output file was root-config.js. This makes the two consistent.